app/controllers/lti_deployments_controller.rb

```diff
@@ -150,6 +159,12 @@ def public_jwk
     render json: { keys: [jwk.export] }
   end
 
+  def course_not_set_up
+    @title = I18n.t('lti.course_not_found')
```

Comment:
I don't think these need to be instance variables. Instead, you can put these strings directly in the `course_not_set_up` partial